http/org_service.go

```diff
@@ -534,7 +534,7 @@ func (h *OrgHandler) handleDeleteSecrets(w http.ResponseWriter, r *http.Request)
 
 type deleteSecretsRequest struct {
 	orgID   influxdb.ID
-	secrets []string
+	Secrets []string `json:"secrets"`
```

Comment:
Does this fit with the swagger definition?

Reply:
Yeah, this ticket also updated the swagger definition.